File: CONEXIONES/GO/POSTGREconex.go
package main

import (
    "database/sql"
    "fmt"
    _ "github.com/lib/pq"
)

func main() {
    connStr := "host=localhost user=postgres password=1234 sslmode=disable"
    db, err := sql.Open("postgres", connStr)
    if err != nil {
        panic(err)
    }
    defer db.Close()

    err = db.Ping()
    if err != nil {
        panic(err)
    }

    fmt.Println("Conexión exitosa")
}

File: CONEXIONES/JAVASCRIPT/POSTGREconex.js
const { Client } = require('pg');

const client = new Client({
    host: 'localhost',
    user: 'postgres',
    password: '1234',
});

client.connect()
    .then(() => {
        console.log("Conexión exitosa");
    })
    .catch(err => {
        console.error("Error de conexión", err.stack);
    })
    .finally(() => {
        client.end();
    });